### DISCUSSION TITLE
(Mac) What should I do after downloading the recommended Java? I tried and it's not working.

### DISCUSSION THREAD

**Discussion by bello89808m** [wcko87/beatoraja-english-guide#39] [beatoraja questions / troubleshooting]
I'm on Mac and I got Liberica Full JRE, downloaded on dmg btw. I then followed the rest of the steps but when I double click beatoraja-config.bat it opens up to visual studio code, and I'm not sure what to do from there. Can I get some help?

**Comment by sigonasr2** [comment]
.bat files only work on Windows. In https://github.com/wcko87/beatoraja-english-guide/wiki#setting-up-beatoraja-with-java it mentions 
`Note: If you are on Mac/Linux, use beatoraja-config.command instead of beatoraja-config.bat.`
Give that a try as that uses proper shell commands.
